### PR TITLE
FEATURE: Add --print-width support using extension setting

### PR DESCRIPTION
**Pull request by xfalcox**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -53,6 +53,11 @@
       "type": "object",
       "title": "Syntax Tree",
       "properties": {
+        "syntaxTree.maximumLineWidth": {
```
Comment (kddnewton) [inline package.json:56]:
```suggestion
        "syntaxTree.printWidth": {
```

File: package.json [hunk 2]
```diff
@@ -53,6 +53,11 @@
...
+          "default": 80,
+          "markdownDescription": "The maximum line width to use when formatting.",
```
Comment (kddnewton) [inline package.json:58]:
```suggestion
          "markdownDescription": "The width to be used when formatting code.",
```

File: src/extension.ts [hunk 3]
```diff
@@ -87,6 +87,9 @@ export async function activate(context: ExtensionContext) {
       args.push(`--plugins=${Array.from(plugins).join(",")}`);
     }
 
+    // Configure print width.
+    args.push(`--print-width=${config.get<number>("maximumLineWidth")}`)
```
Comment (kddnewton) [inline src/extension.ts:91]:
```suggestion
    args.push(`--print-width=${config.get<number>("printWidth")}`)
```

**Review by kddnewton — CHANGES_REQUESTED** [review]
I'd like it to match the option so it's consistent, so `printWidth` for configuration.

**Comment by xfalcox** [comment]
@kddnewton when compiling it locally it's failing to work with 

```
mainThreadExtensionService.ts:111 Activating extension 'ruby-syntax-tree.vscode-syntax-tree' failed: Cannot find module 'semver/functions/parse'
Require stack:
- /home/falco/.vscode-oss/extensions/ruby-syntax-tree.vscode-syntax-tree-0.2.3/node_modules/vscode-languageclient/lib/node/main.js
- /home/falco/.vscode-oss/extensions/ruby-syntax-tree.vscode-syntax-tree-0.2.3/node_modules/vscode-languageclient/node.js
- /home/falco/.vscode-oss/extensions/ruby-syntax-tree.vscode-syntax-tree-0.2.3/out/extension.js
- /usr/lib/code/out/vs/loader.js
- /usr/lib/code/out/bootstrap-amd.js
- /usr/lib/code/out/bootstrap-fork.js.
```

Have you seen something similar?

**Comment by kddnewton** [comment]
@xfalcox nope never seen that. Any chance you need to `yarn install` again?

**Comment by xfalcox** [comment]
Something with my local setup is messed up, and the packaged VSIX node_modules `vscode-languageclient` doesn't have it's own node_modules.

Since vsce depends on semver 5.x and `vscode-languageclient` on 7+, they can't use the shared one and fail to boot. Manually replacing the `vsccode-languageclient` node_modules with the one I have on dev environment fixes the resulting VSIX package and the extension work.

Well, anyway this isn't related to this PR and the code change here works fine, so it's safe to merge and release on marketplace @kddnewton !

**Comment by kddnewton** [comment]
@xfalcox out in the marketplace at 0.3.0

**Pull request merged**